README.md

```diff
@@ -187,6 +188,10 @@ app.getWidget('my-widget');
 
 Each method returns a promise for the respective item. If the item was not registered or could not be loaded, the promise is rejected.
 
+For `widgets`, it is possible for `app` to dynamically create widgets requested via `getWidget()` when the requested `id` has entry in the `defaultWidgetStore` with a `type` attribute that matches a registered `customElememt` name.
+
+**note:** dynamically creating widgets is *only* supported with the registered `defaultWidgetStore`, if there is no registered default store or the state record exists in a non default store the `widget` will not be created.
```

Comment:
Perhaps:

> Widgets may be created dynamically based on state in the default widget store. If the store contains an item for the requested ID, and that item has a `type` attribute that matches a registered custom element name, the requested widget will be created using the custom element factory. This only happens if there was no registered instance or factory for the requested ID.

I think this captures the same information. Note that in the rest of the README "widget", "default widget store" and "custom element" are not typically marked as code.